packages/go_router_builder/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 1.0.6
+
+- Use path-based deps in example
```

Comment:
This needs to follow https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#changelog-style

Reply:
The verb form is still incorrect; it should be "Uses".